jest.config.js

```diff
@@ -1,5 +1,7 @@
+const { FULL_TEST } = process.env;
+
 module.exports = {
-  collectCoverage: true,
+  collectCoverage: !!FULL_TEST,
```

Comment:
related to my other comment below; not a huge fan of the `!!` operator

Reply:
Solved